tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -32,73 +33,21 @@ Check Compose Logs
     Should Not Contain  ${output}  bad address 'aaa'
 
 *** Test Cases ***
-Compose basic
-    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
-
-    Run  echo '${yml}' > basic-compose.yml
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network create vic_default
-    Log  ${output}
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml create
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml start
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml logs
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml stop
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-
-Compose kill
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml up -d
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml kill redis
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml down
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-Compose Up while another container is running (ps filtering related)
-    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d busybox /bin/top
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml up -d
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-Compose Up with link
-    Run  echo '${link-yml}' > link-compose.yml
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file link-compose.yml up -d
-    Log  ${output}
+Compose up in foreground (attach path)   
+    Run  echo '${hello-yml}' > hello-compose.yml
+    ${rc}  ${output}=  Run And Return Rc And Output  mkfifo /tmp/fifo
```

Comment:
Yeah, I should remove it.  I changed the compose yml so I don't need it anymore.